src/gui/clips/ClipView.cpp

```diff
@@ -916,7 +918,8 @@ void ClipView::mouseMoveEvent( QMouseEvent * me )
 			auto sClip = dynamic_cast<SampleClip*>(m_clip);
 			auto pClip = dynamic_cast<PatternClip*>(m_clip);
 			auto aClip = dynamic_cast<AutomationClip*>(m_clip);
-			if( sClip || pClip || aClip )
+			auto mClip = dynamic_cast<MidiClip*>(m_clip);
+			if( sClip || pClip || aClip || mClip )
```

Comment:
There are only 4 types of clip so this condition is now always true. It looks like the only type of clip with special behavior here is pattern clips, so you could keep `pClip` and remove the other 3 variables.